### PR TITLE
docs: update configuration:applications

### PR DESCRIPTION
**Pull request by Jcahilltorre**
Update the configuration doc as per #178 

image

### CODE REVIEW

File: source/configuration/index.rst [hunk 1]
```diff
@@ -3298,6 +3298,16 @@ shared between all application languages:
...
         environment variables to be passed to the app.
 
+        Unit passes the environment variables to the app without modification,
+        even if no enviroment configuration object is specified.
+
+        Any data specified in the **environment** object gets merged to the
+        existing environment variables.
+
+        If an environment variable already exists in the system and gets declared
+        in the object, the object's value takes precendence and gets passed to
```
Comment (javorszky) [inline source/configuration/index.rst:3308]:
```suggestion
        in this object, the object's value takes precedence and gets passed to
```

Comment (javorszky) [reply source/configuration/index.rst:3308]: Besides the two changes, it's g2g

Comment (Jcahilltorre) [reply source/configuration/index.rst]: for this kind of typo and small changes, feel free to apply them directly, no need for my re-review. Ty

Comment (javorszky) [reply source/configuration/index.rst]: Will do in the future, thank you 🙂

**Pull request merged**